types/react-video-recorder/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for react-video-recorder 3.19
+// Project: https://github.com/fbaiodias/react-video-recorder#readme
+// Definitions by: Gabriel Cartier <https://github.com/GabrielCartier>
+//                 Antonio Herrera <https://github.com/aherrera9>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+import { ReactElement } from 'react';
+
+export enum VideoRecorderMimeTypes {
+    WEBM_VP8_OPUS = 'video/webm;codecs="vp8,opus"',
+    WEBM_H264 = 'video/webm;codecs=h264',
+    WEBM_VP9 = 'video/webm;codecs=vp9',
+    WEBM = 'video/webm',
+    MP4 = 'video/mp4'
+}
```

Comment:
```suggestion
export type VideoRecorderMimeTypes =
    | 'video/webm;codecs="vp8|opus"'
    | 'video/webm;codecs=h264'
    | 'video/webm;codecs=vp9'
    | 'video/webm'
    | 'video/mp4';
```
this will serve same purpose. Enums are sometimes overkill in TS/JS:
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums
also:
https://github.com/fbaiodias/react-video-recorder/blob/main/src/video-recorder.js#L22